app/femr/ui/views/history/indexEncounter.scala.html

```diff
@@ -177,15 +177,32 @@
                             <span class="value">@outputStringOrNA(viewModelPharmacy.getProblems.get(problemIterator - 1))</span></p>
                     }
 
-                    @for(prescription <- viewModelPharmacy.getPrescriptions){
-                        <p>
-                            <span class="infoLabel">Dispensed Medication:</span>
-                            @prescription.getAmount @outputStringOrNA(prescription.getName) (@outputStringOrNA(prescription.getMedicationForm).toString().trim)
-                            @for(activeDrug <- prescription.getMedicationActiveDrugs) {
-                                <span class='activeDrug'> @activeDrug.getName @activeDrug.getValue@activeDrug.getUnit </span>
-                            }
+                    @* AJ Saclayan Prescribed Medications *@
+                    @if(viewModelPharmacy.getOriginalMedications.size != null){
+                       <p> <span class="infoLabel">Originally Prescribed Medication:</span></p>
+                        @for(x <- 1 to viewModelPharmacy.getOriginalMedications.size -1) {
+@*//                           defining(viewModelPharmacy.getOriginalMedications.get(x - 1)) { medication =>*@
 
-                        </p>
+                            <p> <i>@viewModelPharmacy.getOriginalMedications.get(x-1).getName</i></p>
+
+                            <p> Replaced with: </p>
+                            <p>@viewModelPharmacy.getOriginalMedications.get(x-1 +1).getName</p>
```

Comment:
I think this just repeats a prescription. If i prescribe 3 medications (without dispensing/replacing any in pharmacy), this is what i get:

![prescribed](https://cloud.githubusercontent.com/assets/5403109/11549326/1a7f5b66-9933-11e5-98c6-74a6223ef98c.png)

Which makes sense because 'getOriginalMedications()' is just a list of prescribed medications that have the potential to be replaced (or are replaced) requested with 'searchService.retrieveReplacedPrescriptionItems(encounterId)' in the history controller.

I'm a bit concerned about repeating the dispensed medication twice (it took me a few to figure out what was going on) - once in the 'originally prescribed medication' section and again in the 'dispensed medication' section. Perhaps the sections could be combined and you could use strike-through text for prescriptions that were replaced? I'm open to ideas here.